app/views/candidate_mailer/reference_received.text.erb

```diff
@@ -1,6 +1,10 @@
 # You have a reference from <%= @reference.name %>
 
-<% if @application_form.enough_references_have_been_provided? %>
+<% if FeatureFlag.active?(:reference_selection) && @application_form.selected_enough_references? %>
+You’ve selected 2 references to send with your application already, but you can change your selection if you want.
+
+Manage your references:
+<% elsif @application_form.enough_references_have_been_provided? %>
```

Comment:
I think we also need to cover this case:

` if FeatureFlag.active?(:reference_selection) && @appliction_form.minimum_references_available_for_selection?`